rules.mk

```diff
@@ -30,6 +30,7 @@ SRC_DIRS := cmd pkg
 
 ALL_ARCH := amd64 arm arm64 ppc64le s390x
 BASEIMAGE ?= k8s.gcr.io/debian-base-$(ARCH):v2.0.0
+IPTIMAGE ?= k8s.gcr.io/debian-iptables-$(ARCH):v12.0.1
```

Comment:
```suggestion
IPTIMAGE ?= us.gcr.io/k8s-artifacts-prod/build-image/debian-iptables-$(ARCH):v12.1.0
```

Reply:
By the time we have an image with iptables 1.8.5, we might be able to use k8s.gcr.io again ;)

Reply:
Potentially, but we don't have a set ETA for the cutover at the moment, so it's better than this picks up an updated image from the new endpoint in the meantime.

A similar change for debian-base just merged in https://github.com/kubernetes/dns/pull/380.

Reply:
I'll leave this suggestion open to pick the right one when this PR is really ready